resqpy/multiprocessing/wrappers/blocked_well_mp.py

```diff
@@ -0,0 +1,133 @@
+"""Multiprocessing wrapper functions for the well/BlockedWell class."""
+
+import logging
+
+log = logging.getLogger(__name__)
+
+from typing import Tuple, Union, List
+from resqpy.model import new_model, Model
+from resqpy.grid import RegularGrid
+from resqpy.well import BlockedWell, Trajectory
+from resqpy.multiprocessing import function_multiprocessing
+from pathlib import Path
+from uuid import UUID
+import uuid
+
+
+def blocked_well_from_trajectory_wrapper(
+    index: int,
+    grid_epc: str,
+    grid_uuid: Union[UUID, str],
+    trajectory_epc: str,
+    trajectory_uuids: List[Union[UUID, str]],
+) -> Tuple[int, bool, str, List[Union[UUID, str]]]:
+    """Wrapper function of the BlockedWell initialisation from a Trajectory.
+
+    Used for multiprocessing to create a new model that is saved in a temporary epc file
+    and returns the required values, which are used in the multiprocessing function to
+    recombine all the objects into a single epc file.
+
+    Args:
+        index (int): the index of the function call from the multiprocessing function.
+        grid_epc (str): epc file path where the grid is saved.
+        grid_uuid (UUID/str): UUID (universally unique identifier) of the grid object.
+        trajectory_epc (str): epc file path where the trajectories are saved.
+        trajectory_uuids (List[UUID/str]): a list of the trajectory uuids used to create each
+            Trajectory object.
+
+    Returns:
+        Tuple containing:
+
+            - index (int): the index passed to the function.
+            - success (bool): True if all the BlockedWell objects could be created, False
+              otherwise.
+            - epc_file (str): the epc file path where the objects are stored.
+            - uuid_list (List[UUID/str]): list of UUIDs of relevant objects.
+    """
+    uuid_list = []
+    tmp_dir = Path(f"tmp_dir/{uuid.uuid4()}")
+    tmp_dir.mkdir(parents = True, exist_ok = True)
+    epc_file = f"{tmp_dir}/wrapper.epc"
```

Comment:
Better to use os.sep than /
Or os.path.join()
Or Path equivalent and str()

Reply:
Yeah I can change that